test/site/_filters/capitalize.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const capitalize = require('../../../src/site/_filters/capitalize');
```

Comment:
do you know if we only use `capitalize` inside of njk templates? I realized [nunjucks has a built-in filter for capitalize](https://mozilla.github.io/nunjucks/templating.html#capitalize) that we could be using. So if we're not using `capitalize` in any of our .js files we could remove it?

Reply:
We use it mainly in our .js files in custom components.

Reply:
We use it mostly in out js files for. ustom component, not njk templates.